users/views.py

```diff
@@ -51,7 +51,10 @@ def UserCreateConfirmView(request, token):
         user.is_active = True
         user.save()
 
-        # TODO redirect('login')
+        # TODO redirect after confirmation:
+        # to profile if logged in
+        # to login if not logged in
+        # to signup if invalid token etc
```

Comment:
Take a look at the `except ObjectDoesNotExist` this is exactly what you have written here.